### DISCUSSION TITLE
Disable extensions by default

### DISCUSSION THREAD

**Discussion by dawaltconley** [neoclide/coc.nvim#4190] [Q&A] [answered]
I'm trying to set up a config where I have most of my coc extensions disabled by default, and only enabled on a per-project basis using a project config / vimrc.

My global config looks like this:

```json
{
  "diagnostic.displayByAle": true,
  "suggest.noselect": true,
  "tsserver.enable": false,
  "eslint.enable": false,
  "html.enable": false,
  "css.enable": false,
  "json.enable": true,
  "tailwindCSS.enable": true,
  "tailwindCSS.includeLanguages": {
    "jinja": "nunjucks"
  },
  "coc.source.OmniSharp.enable": true,
  "coc.source.OmniSharp.triggerCharacters": "."
}
```

I expected e.g. `"tsserver.enable": false` to disable coc-tsserver globally, but when I run `CocList extensions` from any project it still appears enabled.

Am I missing something or is something going wrong here? Thanks.

**Comment by chemzqm** [comment, marked as answer]
`"tsserver.enable": false` disable running of tsserver service.
Use disable action in `:CocList extensions`, use `<tab>` to get action list.
You should `"tsserver.enable": false` if you need tsserver started for some of your project, use `:CocList services` to check status of services.

**Reply by dawaltconley** [reply]
Thank you. Is there no way to configure this so it happens automatically per-project? Something I can put in my vimrc or coc-settings file?

**Reply by chemzqm** [reply]
Use folder configuration `:h coc-configuration`
